osbuild/buildroot.py

```diff
@@ -257,9 +260,40 @@ def run(self, argv, monitor, stage_timeout=None, binds=None, readonly_binds=None
             api_path = "/run/osbuild/api/" + api.endpoint
             mounts += ["--bind", api.socket_address, api_path]
 
+        index = osbuild.meta.Index(self._libdir)
+        def parse_manifest(path):
+            if path == "-":
+                manifest = json.load(sys.stdin)
+            else:
+                with open(path) as f:
+                    manifest = json.load(f)
+
+            return manifest
+        
+        desc = parse_manifest("test/data/manifests/fedora-ostree-image.json")
```

Comment:
This needs to be replaced with the passed in manifest, I was just debugging here.